### ISSUE TITLE
Sometimes, there's not much more to write beyond the short description

### ISSUE THREAD

**Issue by annegentle** [russell/fairy-slipper#41]
I see a lot of " Unfortunately this API operation is not fully documented." when there's not much more to say about a particular call. :no_mouth: Heh. So should we just move that one-liner short description into that area?

**Comment by annegentle** [comment]
Oh, and I'm also seeing the opposite, where GET "List extensions" for the Networking API has "None" for the short description and then has text when you expand the call. So we should probably call the short description good enough for one of the places.

**Comment by russell** [comment]
Yeah, i think that the "Unfortunately this API operation is not fully documented." should probably be replaced with an image.  So that the page formatting looks ok even when there is no text.  Yeah, we shouldn't have any `Nones` creeping through.  Thats bad.

**Comment by annegentle** [comment]
I'm not a fan of an image -- seems shame based and we won't get the shame from the people who write this stuff since they don't read it, and it makes for a worse experience for the readers. How about repeating the shortdesc but then littering the source with comments like "THIS NEEDS MOAR DOC PLZ"?
